### ISSUE TITLE
Make it possible to set search provided URL

### ISSUE THREAD

**Issue by e-gaulue** [GoToTags/GoToTags-iPhone-App#7]
It presently auto-search on Google. Would be far better if we could provide an URL in order to work with our websites.

**Comment by ctadlock** [comment]
That is another feature we will be adding; the ability to add a custom url "action" that can either be automatically performed or performed with a button click. That url will be able to be dynamic; `http://mydomain.com?id={{tag.uid}}`

That work?

**Comment by e-gaulue** [comment]
Yes, `'http://mydomain.com?id='+urlencode({{tag.uid}})` would even be better.

I need it for next Thuesday, maybe I should look for something else in the meanwhile. But I like your app that as a Apple spirit.

Regards,

**Comment by ctadlock** [comment]
Unfortunately it wont be available by next week. I doubt Apple could even approve an app that fast.

**Comment by e-gaulue** [comment]
Well, in the meanwhile, I will connect the iPhone to my network through VPN and then I'm the Oil King being able to do whatever I wan't as redirecting request to google to my own server. Just crazy all those things we are able to do nowadays.
